src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1957,7 +1995,12 @@ namespace UnitarySystems {
                 SizingMethod = DataHVACGlobals::AutoCalculateSizing;
                 this->m_MaxNoCoolHeatAirVolFlow = DataSizing::AutoSize;
             } else {
-                DataSizing::DataFractionUsedForSizing = this->m_NoLoadAirFlowRateRatio;
+                if (this->m_CoolingCoilType_Num == DataHVACGlobals::Coil_CoolingAirToAirVariableSpeed ||
+                    this->m_HeatingCoilType_Num == DataHVACGlobals::Coil_HeatingAirToAirVariableSpeed) {
+                    DataSizing::DataFractionUsedForSizing = NoLoadFractionForVarSpeedCoils;
+                } else {
+                    DataSizing::DataFractionUsedForSizing = this->m_NoLoadAirFlowRateRatio;
+                }
```

Comment:
Then you don't need this if block

Reply:
I do not need this block if I am not using this variable for the other sizing methods. Removed.